### PR TITLE
10nowv 12879

### PR DESCRIPTION
**Pull request by ijobling**

### CODE REVIEW

File: source/common/develop/ide/editing/force-language-highlighting.rst [hunk 1]
```diff
@@ -10,5 +10,12 @@ To choose the language to use for syntax highlighting, click the language in the
 
 .. figure:: /img/ide-force-lang.png
    :alt: Force Language Highlighting
+   
+ECMAScript 6 support
+--------------------
+
+ECMAScript 6 is enabled by default in javascript files.
+
+If you wish addtional configurations to apply, you can add a `.jshintrc` file to the root of your project/assignment to handle additional configurations. See https://github.com/jshint/jshint/blob/master/examples/.jshintrc for more information
```
Comment (shajason) [inline source/common/develop/ide/editing/force-language-highlighting.rst:19]: Typo on line 19 - additional.

File: source/instructors/admin/organization/sso.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    
+5: Save the configuration
+
+6: Go to **Parameters** and set up the Custom User fields, checking **Include in SAML assertion** mapping the value field to the relevant fields from your SSO
+
+    - for **Role**, set the value to **Role(Custom)**
+    
+7: Go to **SSO** and copy the **SAML 2.0 Endpoint HTTP** and paste into the **Identity Provider URL** field in Codio, and find and copy the **X. 509 Certificate** into the **Certificate** field in Codio.
+
+.. Note:: These field names relate to onelogin and may be different in other Service Providers
```
Comment (shajason) [inline source/instructors/admin/organization/sso.rst:39]: Line 39 - OneLogin should be cased as I just wrote and maybe bolded or italicized to indicate it is a product (not sure what you usually do). Service Providers does not need caps.

File: source/instructors/setupcourses/assignment-settings/exam-proctoring.rst [hunk 3]
```diff
@@ -46,4 +46,4 @@ The Exam Proctoring settings are used to enable various options for assignment e
 
   On the **Students** tab in the course, click the menu icon (3 horizonatal dots) and choose **Reset Single Login**.
 
-
+- **Authentication** - If enabled, your students will need to authenticate before they can open the assignment. This option can only be used if you have configured :ref:`SSO Integration <sso-integration>` for the organisation.
```
Comment (shajason) [inline source/instructors/setupcourses/assignment-settings/exam-proctoring.rst:49]: Line 49: organisation -> organization

Can we add a word or two here to explain what this authentication means - how does it support the exam process - I truly am unclear on this.

Comment (ijobling) [reply source/instructors/setupcourses/assignment-settings/exam-proctoring.rst]: Far as I read it all, with this enabled no matter how they logged into Codio (directly, through LMS or through SSO provider) they have to authenticate again before starting the assignment (ie enter email/pwd etc)

**Review by shajason — APPROVED** [review]

**Pull request merged**